site/src/atomics/flex.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Flex Atomics
+description: Flex related atomic css classes for the Atlas Design System
+template: standard
```

Comment:
We've been using tables for many of our Atomics to help keep them organized. This one will be a larger one, but here's a small example:

```md
| cssproperty       | value                                                                             | screensize |
| ----------------- | --------------------------------------------------------------------------------- | ---------- |
| `flex-direction`  | `column`, `row-reverse`, `column-reverse`                                         | `tablet`   |
| `justify-content` | `flex-start`, `flex-end`, `flex-start`, `center`, `space-around`, `space-between` | `tablet`   |
```

Reply:
See (`atomics.template`)

Reply:
updated